res/xml/bone.xml

```diff
@@ -0,0 +1,45 @@
+<?xml	version="1.0"	encoding="utf-8"?>
+<keyboard>
+	<row>
+		<key   width="1"   key0="j"							key2="°"						key4="esc"/>
+		<key   width="1"   key0="d"							key2="§"						key4="_"/>
+		<key   width="1"   key0="u"							key2="ℓ"						key4="["/>
+		<key   width="1"   key0="a"							key2="»"						key4="]"/>
+		<key   width="1"   key0="x"							key2="«"						key4="^"/>
+		<key   width="1"   key0="p"			key1="$"						key3="!"		key4="7"/>
+		<key   width="1"   key0="h"			key1="€"		key2="⟨"		key3="&lt;"		key4="8"/>
+		<key   width="1"   key0="l"			key1="£"		key2="⟩"		key3="&rt;"		key4="9"/>
```

Comment:
Are `⟨` and `⟩` common ? The `Fn` layer starts to fill up but I'd suggest `Fn+;` and `Fn+:`.